Mobile Buy SDK/Mobile Buy SDK/Data/BUYClient+Routing.h

```diff
@@ -36,13 +36,13 @@
 - (NSURL *)urlForCollectionListingsWithParameters:(NSDictionary *)parameters;
 
 - (NSURL *)urlForCheckouts;
-- (NSURL *)urlForCheckoutsWithToken:(NSString *)token;
-- (NSURL *)urlForCheckoutsProcessingWithToken:(NSString *)token;
-- (NSURL *)urlForCheckoutsCompletionWithToken:(NSString *)token;
-- (NSURL *)urlForCheckoutsShippingRatesWithToken:(NSString *)token parameters:(NSDictionary *)parameters;
+- (NSURL *)urlForCheckoutsForToken:(NSString *)token;
```

Comment:
These method signatures sound bizarre. (They did before, too.) Is this functionality private? I don't want to get hung up on this, but would like to be able to change this later. Would really like `-urlForCheckoutToken:` etc. but don't know if that's inaccurate.